packages/cli/generators/discover/index.js

```diff
@@ -171,6 +181,7 @@ module.exports = class DiscoveryGenerator extends ArtifactGenerator {
     ];
 
     return this.prompt(prompts).then(answers => {
+      /* istanbul ignore next */
```

Comment:
I found multiple ignores added, are they necessary?

Reply:
CLI files are a bit different from others. We don't really test the CLI themselves. And the coverage would complain about the not covered if statements in these files. 
I am okay to remove them just that the coverage test would fail.